kedro/pipeline/pipeline.py

```diff
@@ -380,6 +381,18 @@ def only_nodes(self, *node_names: str) -> "Pipeline":
         """
         unregistered_nodes = set(node_names) - set(self._nodes_by_name.keys())
         if unregistered_nodes:
+            # check if unregistered nodes are available under namespace
+            for name in node_names:
+                namespaces = [
+                    k
+                    for k, v in self._nodes_by_name.items()
```

Comment:
It would be helpful to give more meaningful names to these variables, to make it easier to read the code.

Reply:
Done!